drivers/usb/udc/udc_mcux_ehci.c

```diff
@@ -20,9 +20,7 @@
 #include "usb_device_config.h"
 #include "usb_device_mcux_drv_port.h"
 #include "usb_device_ehci.h"
-#ifdef CONFIG_DT_HAS_NXP_USBPHY_ENABLED
```

Comment:
Is this just some rebase artifact? This commit doesn't seem to belong to this PR.

Reply:
I needed it for testing but it was already merged when I pushed.